string[] names
float32[] targetAngles
float32 maxSpeedFraction
---
bool res